src/dashboard/main.py

```diff
@@ -6,12 +6,14 @@
 
 app = Dash(__name__, use_pages=True, external_scripts=external_scripts)
 
+PORT = 3030
+
 
 def page_container() -> Component:
-    return html.Div([html.H1("PUM13-2023"), dash.page_container])
+    return html.Div(children=[dash.page_container])
 
 
 app.layout = page_container
 
 if __name__ == "__main__":
-    app.run("127.0.0.1", "8000")
+    app.run("127.0.0.1", PORT, debug=True)
```

Comment:
Varför byter vi port? Känns aningen out-of-scope för branchen.
Går det att lägga navbar en page containern iställlet?
Vet inte om det finns något bra sätt att ta in namnet på nuvarande sidan, men om det går så skulle det bli snyggare så.

Reply:
Tror jag gjorde om den till konstant istället men glömde att sätta till samma port

Reply:
Vad ligger bakom valet av port 3090? Ser ut som det ändrades i 5e979fe30a0745a07beac6e7d59321d2ef662ae4 .

Reply:
Tror det var helt random